bin/cron/check_for_census_inaccuracy_reports

```diff
@@ -5,7 +5,10 @@ require_relative '../../dashboard/config/environment'
 count = Census::CensusSubmission.unresolved_reported_inaccuracies.count
 
 if count > 0
-  result = ChatClient.message 'access_report', "There are #{count} reported inaccuracies to review. Somebody with census_review permissions should go to https://studio.code.org/census/review to look at them."
+  message = count == 1 ? "There is 1 reported inaccuracy to review. " : "There are #{count} reported inaccuracies to review. "
```

Comment:
Could we use a generic message?  `"There are 1 or more inaccuracy entries to review (report count - #{count}):" `

Reply:
We could but I wanted to make this more chatty since it shows up in our channel and is a bot :)

Reply:
There are 1 product managers who approve this PR